### PR TITLE
[SDPA-3820] Update icon config

### PR DESCRIPTION
**Pull request by GROwen**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-3820

### Related PRs
dpc-sdp/tide_core#128
dpc-sdp/tide_site#50

### Changed
1. Fixed embed media icon broken link

### CODE REVIEW

File: tide_media.install [hunk 1]
```diff
@@ -381,3 +354,14 @@ function tide_media_update_8009() {
...
...8ZXhpZjpDb2xvclNwYWNlPjE8L2V4aWY6Q29sb3JTcGFjZT4KICAgICAgICAgPGV4aWY6UGl4ZWxZRGltZW5zaW9uPjE4PC9leGlmOlBpeGVsWURpbWVuc2lvbj4KICAgICAgICAgPGRjOnN1YmplY3Q+CiAgICAgICAgICAgIDxyZGY6U2VxLz4KICAgICAgICAgPC9kYzpzdWJqZWN0PgogICAgICAgICA8eG1wOk1vZGlmeURhdGU+MjAxNToxMToxMCAxMjoxMTo0NDwveG1wOk1vZGlmeURhdGU+CiAgICAgICAgIDx4bXA6Q3JlYXRvclRvb2w+UGl4ZWxtYXRvciAzLjQ8L3htcDpDcmVhdG9yVG9vbD4KICAgICAgPC9yZGY6RGVzY3JpcHRpb24+CiAgIDwvcmRmOlJERj4KPC94OnhtcG1ldGE+CsT0zU0AAAIVSURBVDgRjVNNS2JhFPbeq04t/BpoEZOBJWIkrVwYs8gfMLgwSAh3AxYxEbQJWiVBi3ZBILNtZikym1m7VnAxQ5SgG1cuxK8SB7/neS694o3rrQPH93w853nPPe9RMb0hPp9vzQ5ptVptI6hslIxEIrbxeJyWJOlnOBxeMMIqRkmLxbI3GAyOQLTa7/f/1Gq1h3n4uR0lEglLr9dLiEIQHTMm/NenUUe73W73FAUSiyaTibvZbN5Xq1XdrqZEAEqNRmMJNSvBYDBSr9ev0YWdJC8imc3mz6FQqOlwOLrRaFTO5XL/ksmkmpa8Xu+m1Wr9hllsyLLsHY1GdpDaoIJAc2JeJuizoihPeIgy5viIC2/NMLYxi0OiQaIp0nN4AS8CiQ2En1C7g0YKMlq7w5qk9IreE3M6nal0On2nDrJUKn2IxWI37Xb7YN4nvSblJ2JW3wuFwgnsnkpEEAgUv99/MRwOz2HPXQtiUTjG4K+KxeIFbHUe0wIGMpnMJWam+7wkEEIMsYKE8SkRnXw+L2OIb06cGGJZI2S6Rwxg4ZY7nc4ZgIsCoHdiTazlcvlHpVJ5EnkNKwDrWAGnSKJ1E2Kq0hZCDLHC56khQjdrKFBjLIT+crvdX6i0GaPglIlVnZcfDRE+7SOfHwv21+Vy7cfj8d1sNvubSpsx5oghdpZIY3s8nq1AIPAV/3KHJjHjMEcMsTNh038wIuU96QNsXAAAAABJRU5ErkJggg==');
```
Comment (anthony-malkoun) [inline tide_media.install]: Is there no other way to do this? Is it just a base 64 encode?

Comment (GROwen) [reply tide_media.install]: Yeah it's a base64 encoded icon.

Is it that it **is** a base64 encoded icon that you don't like or the method of updating the value?

You question does make me realise that the image file is still part of the repo, which I'll push a new commit to remove.

**Comment by anthony-malkoun** [comment]
I think you need to pull Vincent's change in to get rid of the broken Gherkin patch.

File: tide_media.install (hunk 1 above)
Comment (anthony-malkoun) [reply tide_media.install]: Ah, it's a minor thing... I was hoping there would be something like...

```
$media_config->set('icon.data', base64_encode(file_get_contents('star.png'));
```
 But it still doesn't do anything for cleaning the string out of the YAML. Apparently you can write your own loaders for Symfony YAML but we won't be doing that :)

**Pull request merged**